benchmarks/test_startup_time.py

```diff
@@ -0,0 +1,66 @@
+from wasmer import engine, Store, Module, Instance
+from wasmer_compiler_cranelift import Compiler as Cranelift
+from wasmer_compiler_llvm import Compiler as LLVM
+from wasmer_compiler_singlepass import Compiler as Singlepass
+
+TEST_BYTES = open('benchmarks/nbody.wasm', 'rb').read()
+
+def test_benchmark_startup_time_nbody_cranelift_jit(benchmark):
```

Comment:
We should rename those benchmarks `headless` rather than `startup` I guess. Thoughts?